apps/nextjs-website/src/app/profile/personal-data/page.tsx

```diff
@@ -14,37 +15,48 @@ const PersonalData = () => {
     },
   } = translations;
   const t = useTranslations('profile');
-  const { user } = useUser();
+  const { user, setUserAttributes } = useUser();
 
-  const dataSectionItems: InfoCardItemProps[] = [
-    {
-      title: t('personalData.fields.name'),
-      value: user?.attributes.given_name,
-    },
-    {
-      title: t('personalData.fields.surname'),
-      value: user?.attributes.family_name,
-    },
-    {
-      title: t('personalData.fields.role'),
-      value: user?.attributes['custom:job_role'],
-    },
-    {
-      title: t('personalData.fields.sector'),
-      value: companyRoles.find(
-        (role) => role.value === user?.attributes['custom:company_type']
-      )?.title,
-    },
-  ];
+  const [dataSectionItems, setDataSectionItems] = useState<InfoCardItemProps[]>([]);
+
+  useEffect(() => {
+    setDataSectionItems(
+      [
+        {
+          title: t('personalData.fields.name'),
+          value: user?.attributes.given_name,
+          editable: true,
+        },
+        {
+          title: t('personalData.fields.surname'),
+          value: user?.attributes.family_name,
+          editable: true,
+        },
+        {
+          title: t('personalData.fields.role'),
+          value: user?.attributes['custom:job_role'],
+          editable: true,
+        },
+        {
+          title: t('personalData.fields.sector'),
```

Comment:
This field should be e Select like in the signu up form
![image](https://github.com/pagopa/developer-portal/assets/39835990/182d6e77-47b2-40b2-9cf0-79a5735ccdb7)
Figma:
![image](https://github.com/pagopa/developer-portal/assets/39835990/89b0c2e3-3c26-457a-a75b-e6ee5c23db99)